_data/projects/eBooker.yml

```diff
@@ -1,11 +1,13 @@
 name: eBooker
-desc: eBooker is a command-line application written in Python. With it, you don't have to learn programming or manage massive user interfaces to make great eBooks. We provide simple download scripts for all major operating systems, so just download, install, and enjoy.
+desc: eBooker is a command-line tool which lets you create ebooks and other text files from command line with ease. You don't have to be a programming expert or a nerd to use this. Anyone with a basic knowledge in computers can use this tool very easily. We provide just one script to run, so run it! So far, it can execute simple commands and also create and edit a file. Booker is in development, though It is by far not finished! It has a long way to go, but please help us make it! If you want to develop on it (providing Git is installed) open up your terminal and type in git clone https://github.com/arch-master/eBooker.git --branch development --single-branch.
```

Comment:
Let's drop this bit:

> Booker is by far not finished! It has a long way to go, but please help us make it!

Because you're already listed on the site, I think it's implied that you would like assistance.